<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseInstance</class>
 <widget class="QDialog" name="CourseInstance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>560</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CHECK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="header">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="quitButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="heart1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="heart2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="heart3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="score"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="taskSlot" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>621</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>CourseInstance</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
